examples/dom-bindings/src/App.js

```diff
@@ -23,32 +23,34 @@ class App extends Component {
           <h2>Welcome to React PDF</h2>
         </div>
         <p className="App-intro">
-          <Document height="100%" width="100%">
-            <Page size="A4">
-              <View style={styles.container}>
-                <Text>
-                  Text
-                </Text>
-                <View style={styles.block}>
+          <Pdf height="100%" width="100%">
+            <Document>
```

Comment:
Yeah was also thinking about doing that, then the specific platform component would just render the Document internally, will change it ASAP.